cmd/scollector/collectors/dsc_windows.go

```diff
@@ -0,0 +1,63 @@
+package collectors
+
+import (
+	"os"
+	"time"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+func init() {
+	collectors = append(collectors, &IntervalCollector{F: c_dsc_mof, Interval: time.Minute * 5})
+}
+
+const (
+	dscLCM = "dsc.lcm."
+	dscMof = "dsc.mof."
+)
+
+var (
+	dscpath     = os.ExpandEnv(`${SYSTEMROOT}\system32\Configuration\`)
+	mapMofFiles = map[string]string{
+		"MetaConfig.mof":       "Meta_Config",
+		"Current.mof":          "Current_Config",
+		"backup.mof":           "Backup_Config",
+		"pending.mof":          "Pending_Config",
+		"DSCStatusHistory.mof": "DSC_History",
+		"DSCEngineCache.mof":   "DSC_Cache",
+	}
+)
+
+// c_dsc_mof monitors the size and last modified time of each mof file.
+// These out of band metrics can be used to verify the DSC WMI Status metrics.
+func c_dsc_mof() (opentsdb.MultiDataPoint, error) {
+	var md opentsdb.MultiDataPoint
+	c := 0
+	if _, err := os.Stat(dscpath + "MetaConfig.mof"); os.IsNotExist(err) {
+		c = 1
+	}
+	Add(&md, dscLCM+"configured", c, nil, metadata.Gauge, metadata.StatusCode, descDSCLCMConfigured)
+	if c == 1 {
+		return md, nil
+	}
+	for filename, filetype := range mapMofFiles {
+		tags := opentsdb.TagSet{"type": filetype}
+		fi, fierr := os.Stat(dscpath + filename)
+		s := int64(-1)
```

Comment:
That is the "master file", so if it doesn't exist the LCM doesn't exist or isn't configured (like a linux system not running puppet). If that file exists, then we need to track these files, and I'd rather not use a separate "exist" metric for each. I don't see any issues with using -1, as even with aggregations it will make sense (-1 will act just like 0, which is conceptually what I would expect)